WORKSPACE

```diff
@@ -330,9 +324,26 @@ http_file(
 ## Protobuf codegen rules
 ##
 
-load("@io_bazel_rules_go//proto:go_proto_library.bzl", "go_proto_repositories")
+# Note: do not use go_proto_repositories since it has old versions of protobuf and grpc-go
+
+new_go_repository(
+    name = "com_github_golang_protobuf",
+    commit = "8ee79997227bf9b34611aee7946ae64735e6fd93",
+    importpath = "github.com/golang/protobuf",
+)
+
+http_archive(
+    name = "com_github_google_protobuf",
+    sha256 = "2a25c2b71c707c5552ec9afdfb22532a93a339e1ca5d38f163fe4107af08c54c",
+    strip_prefix = "protobuf-3.2.0",
+    url = "https://github.com/google/protobuf/archive/v3.2.0.tar.gz",
+)
 
-go_proto_repositories()
+new_go_repository(
+    name = "org_golang_google_grpc",
+    importpath = "google.golang.org/grpc",
+    tag = "v1.2.1",
```

Comment:
Why the switch to "tag" instead of "commit' here? Weren't we using "commit" even if cases where tag was available because tag wasn't 100% immutable?